app/views/layouts/application.html.erb

```diff
@@ -16,12 +16,7 @@
           <% end %>
       </div>
     </div>
-    <div class="row">
-      <div class="col-xs-12">
-        <div class="alert alert-info" role="alert"><%= notice %></div>
-        <div class="alert alert-danger" role="alert"><%= alert %></div>
-      </div>
-    </div>
+    <%= render partial: "common/flash", flash: flash %>
```

Comment:
You shouldn't need to pass `flash` in... the partial should have access to it already